test/PlaidML/CppEdsl.Dot.mlir

```diff
@@ -0,0 +1,37 @@
+#map0 = affine_map<(d0, d1, d2) -> (d0, d2)>
```

Comment:
please add the RUN comment for pipeline at the top..

Reply:
we are adopting a two-step procedure. first we add the test with input and output.
then we enable the tests one by one.
currently, we get the input/output by handwritten or run by cpu or python code.
run comment of gpu pipeline can be added after your and dimple's patch to verify it.

Reply:
you can add the run statement and disable them from lit for now

Reply:
now added

Reply:
we need for all these tests

Reply:
coming on the way >_<